src/store/loginData.js

```diff
@@ -0,0 +1,63 @@
+import router from "../router";
+import axios from "axios";
+const state = {
+    login: {
+        username: "",
+        password: "",
+    },
+    loginError: "",
+    loginLoading: false,
+    loginRole: ""
+}
+const actions = {
+    loginUser({ dispatch }) {
+        if (state.login.username === "" || state.login.password === "") {
+            state.loginError = "* Fill Required Details";
+        } else {
+            state.loginLoading = true;
+            axios.post(`https://secure-refuge-14993.herokuapp.com/login?username=${state.login.username}&password=${state.login.password}`).then(response => {
+                if (response.status === 200 && response.data.error === 0) {
+                    router.push("/take");
+                    state.login.username = "";
+                    state.login.password = "";
+                    state.loginError = "";
+                    localStorage.token = response.data.token;
+                    state.loginLoading = false;
+                    dispatch('decodeToken');
+                    localStorage.loginRole = state.loginRole;
+                }
+                if (response.status === 200 && response.data.error === 1) {
+                    state.loginError = "* User not exists ";
+                    state.login.username = "";
```

Comment:
make state.login.username & password blank just below the if condition ends. and remove from both if codition